zim/notebook/index/pages.py

```diff
@@ -539,6 +539,17 @@ def walk_bottomup(self, path=None):
 		page_id = self._pages.get_page_id(path) if path else ROOT_ID # can raise
 		return self._pages.walk_bottomup(page_id)
 
+	def search_pagename_substring(self, nameFragment):
+		candidates = self.db.execute(
+			'SELECT * FROM pages WHERE LOWER(name) LIKE ? ORDER BY parent,sortkey',
+			('%'+nameFragment.lower()+'%',)
+		).fetchall();
+
+		logger.debug("%d page-names contain '%s'", len(candidates), nameFragment);
```

Comment:
Would remove these kind of debug messages before merge - they are useful while developing, but later on only add noise to the error log